mne/viz/utils.py

```diff
@@ -168,15 +168,19 @@ def mne_analyze_colormap(limits=[5, 10, 15], format='mayavi'):
 
     """
     l = np.asarray(limits, dtype='float')
```

Comment:
let's avoid these single letter variable names if possible. Chances are high that someone will suffer when using the python debugger.